.github/workflows/cypress.yml

```diff
@@ -17,13 +20,30 @@ jobs:
           - {name: 'WP trunk', version: 'WordPress/WordPress#master'}
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+    - name: Download build zip
+      uses: actions/download-artifact@v3
+      with:
+        name: ${{ github.event.repository.name }}
+        path: ${{ github.event.repository.name }}
+    - name: Display structure of downloaded files
+      run: ls -R
+      working-directory: ${{ github.event.repository.name }}
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-node-modules
+      with:
+        path: |
+          node_modules
+          ~/.cache
+          ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
     - name: Install dependencies
       run: npm install
-    - name: Build (optional)
-      run: npm run build
-    - name: Set the core version
-      run: ./tests/bin/set-core-version.js ${{ matrix.core.version }}
+    - name: Set the core version and plugins config 
+      run: ./tests/bin/set-wp-config.js --core=${{ matrix.core.version }} --plugins=./${{ github.event.repository.name }}
```

Comment:
✅ Verified that running `./tests/bin/set-wp-config.js --core=WordPress/WordPress#master --plugins=./some-directory` produces the following (formatted for readability):

```json
{
	"plugins":["./some-directory"],
	"env":{
		"tests":{
			"mappings":{
				"wp-cli.yml":"./tests/bin/wp-cli.yml"
			}
		}
	},
	"core":"WordPress/WordPress#master"
}
```

See note below though.